examples/full.html

```diff
@@ -0,0 +1,21 @@
+<script>
+  (function(i,s,o,g,r,a,m){
+      i['BarnebysAnalyticsObject']=r;
+      i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();
+      a=s.createElement(o),m=s.getElementsByTagName(o)[0];
+      a.async=1;
+      a.src=g;
+      m.parentNode.insertBefore(a,m)
+  })(window,document,'script','http://localhost:3000/bite.v1.js','ba');
```

Comment:
The path should be `http://local host:3000/v1/bite.js` now that it’s served by `now dev`